### ISSUE TITLE
Trying to install pywinauto on windows 10 x64 Python 3.4

### ISSUE THREAD

**Issue by Sky2hawkZ** [pywinauto/pywinauto#239]
c:\python34\lib\distutils\dist.py:260: UserWarning: Unknown distribution option: 'entry_points'
      warnings.warn(msg)
    c:\python34\lib\distutils\dist.py:260: UserWarning: Unknown distribution option: 'test_suite'
      warnings.warn(msg)
    c:\python34\lib\distutils\dist.py:260: UserWarning: Unknown distribution option: 'zip_safe'
      warnings.warn(msg)
    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
       or: setup.py --help [cmd1 cmd2 ...]
       or: setup.py --help-commands
       or: setup.py cmd --help

    error: invalid command 'bdist_egg'
    AppData\Local\Temp\pip-build-eusjzarn\pywinauto\distribute-0.6.14-py3.4.egg
    Traceback (most recent call last):
      File "c:\python34\lib\site-packages\ez_setup.py", line 143, in use_setuptools
        raise ImportError
    ImportError

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "AppData\Local\Temp\pip-build-eusjzarn\pywinauto\setup.py", line 32, in <module>
        use_setuptools()
      File "c:\python34\lib\site-packages\ez_setup.py", line 145, in use_setuptools
        return _do_download(version, download_base, to_dir, download_delay)
      File "c:\python34\lib\site-packages\ez_setup.py", line 125, in _do_download
        _build_egg(egg, tarball, to_dir)
      File "c:\python34\lib\site-packages\ez_setup.py", line 116, in _build_egg
        raise IOError('Could not build the egg.')
    OSError: Could not build the egg.

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in AppData\Local\Temp\pip-build-eusjzarn\pywinauto\0

**Comment by vasily-v-ryabov** [comment]
Actual `setup.py` in master branch is pretty old. Could you please try `setup.py` from UIA branch?

**Comment by Sky2hawkZ** [comment]
Managed to get Pywinauto going after reinstalling a newer version of Python.

But I'm gonna hijack the thread while I can.

I want to substitute the "pyautogui.typewrite" function if you are aware of it, with something from pywinauto.

pyautogui.typewrite just writes in whatever field is open and writable in at the time of the command processing through Python. 

Is there a similar command for Pywinauto? Because I will have to use Pywinauto in programs that do not support the "app.[program].[programfield].TypeKeys" syntax structure.

To summarize, is there a command in Pywinauto that just writes what is sent into it's function on whatever current field you are currently in?

**Comment by vasily-v-ryabov** [comment]
We have a standalone module `SendKeysCtypes` in pywinauto 0.5.4. It doesn't depend on any control accessibility.
Example:
```python
from pywinauto import SendKeysCtypes as keyboard # will be named just "keyboard" in 0.6.0
keyboard.SendKeys("type something{ENTER}", pause=0.0, with_spaces=True)
```

**Comment by Sky2hawkZ** [comment]
Thank you so much! :) Considered the thread closed then!
